app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt

```diff
@@ -532,14 +537,41 @@ fun CommentItem(
   val networkManager = NetworkManager(context)
 
   Column(modifier = Modifier.padding(STANDARD_PADDING.dp)) {
```

Comment:
display a badge for comments made by the creator, enhancing the UI by clearly indicating the creator's comments.